lib/bb/generate_report_request_form.rb

```diff
@@ -7,7 +7,7 @@ class GenerateReportRequestForm < Common::Form
 
     ELIGIBLE_DATA_CLASSES = %w( seiactivityjournal seiallergies seidemographics
                                 familyhealthhistory seifoodjournal healthcareproviders healthinsurance
-                                seiimmunizations labsandtests medicalevents medications militaryhealthhistory
```

Comment:
medications is a valid class according to the response for a premium user yesterday, so not sure why you removed it. But more to the point, I think this `ELIGIBLE_DATA_CLASSES` array is never used and is misleading to have here. I'm in favor of removing it. If we want a reference of what the eligible data classes typically are, let's add it as a comment and say that it's illustrative only.